lib/matplotlib/lines.py

```diff
@@ -793,16 +793,15 @@ def draw(self, renderer):
             rgbaFace = self._get_rgba_face()
             rgbaFaceAlt = self._get_rgba_face(alt=True)
             edgecolor = self.get_markeredgecolor()
-            if (isinstance(edgecolor, six.string_types)
-                    and edgecolor.lower() == 'none'):
+            if cbook._str_equal(edgecolor, "none"):
```

Comment:
Original code has `edgecolor.lower()`, I do not see any normalization in `set_markeredgecolor`/`get_markeredgecolor`. Did I miss something?

Reply:
fixed